### DISCUSSION TITLE
Warning - messages

### DISCUSSION THREAD

**Discussion by emassen** [mikopbx/Core#492] [General ( Обо всём )]
Hi,

WARNING[7530][C-00000a7d] pbx_functions.c: Can't find trailing parenthesis for function 'DIALPLAN_EXISTS(internal ....

This is normal?

**Comment by boffart** [comment]
See the discussion on this topic
https://qa.mikopbx.ru/14558/странные-сообщения-в-логах
Commit:
https://github.com/mikopbx/ModuleUsersGroups/commit/c16aeb6de49ac361d651df21c6fc6f4d0d71591d
